cgi-bin/DW/Controller/Support/History.pm

```diff
@@ -107,7 +107,7 @@ sub history_handler {
                 $reqs{$row->[0]} = {
                     spid => $row->[0],
                     winner => $points{$row->[0]}->[0],
-                    points => $points{$row->[0]}->[1]+0,
+                    points => $points{$row->[0]}->[1] || 0,
```

Comment:
Yeah the reason I didn't do that here was because the value from the database is typed to be an integer, and I believe the person who wrote the original statement used `+0` when they meant `|| 0`, just to guard against undefs.

So yeah, I had my reasons in the wider context of things! Thanks for giving me the benefit of the doubt :wink:  But if you think there's a stronger argument to be made in the other direction, I'm convincible.